client/ayon_core/hosts/maya/api/fbx.py

```diff
@@ -102,6 +102,7 @@ def default_options(self):
             "constraints": False,
             "lights": True,
             "embeddedTextures": False,
+            "includeChildren": True,
```

Comment:
If adding this - shouldn't this also be in the return of the `options` method?

Reply:
indeed, it should be also in the options